Level-2/trmv/trmv.cpp

```diff
@@ -99,23 +101,22 @@ int main(int argc, char** argv)
     // as rocblas_float_complex so can copy the data into the array for use in the rocblas C API
     std::vector<T> hA(sizeA);
     std::vector<T> hX(sizeX);
+    std::vector<T> hXCopy(sizeX);
     std::vector<T> hXGold(sizeX);
-    std::vector<T> hXGold_work(sizeX);
 
     // initialize uniform random data with lower and upper range
     helpers::fillVectorUniformRealDist(hA, -0.5, 0.5);
     helpers::fillVectorUniformRealDist(hX, -0.5, 0.5);
 
+    hXCopy = hX;
+
```

Comment:
Could we initialize hXGold to zeros?

Reply:
It could be done, for educational value?

Reply:
Thought it would be good to initialize rather than having some random values.

Reply:
Okay gold's original values are not used by your reference but I pushed a zero init.    Setting it to X was confusing me thinking it matched the blas form of updating X which it is not.
In c++ 11 and earlier it likely was already zero init by the default values of the constructor.   Could later on add a PASS/FAIL result on this to match some of the others examples.   Would also add const to the reference to make it clear it is not like blas.